go.mod

```diff
@@ -3,61 +3,24 @@ module github.com/kubernetes-csi/csi-lib-utils
 go 1.16
 
 require (
-	github.com/container-storage-interface/spec v1.2.0
-	github.com/golang/protobuf v1.4.2
-	github.com/stretchr/testify v1.5.1
-	golang.org/x/net v0.0.0-20200707034311-ab3426394381
-	google.golang.org/grpc v1.29.0
-	k8s.io/api v0.19.0
-	k8s.io/client-go v0.19.0
-	k8s.io/component-base v0.19.0
-	k8s.io/klog/v2 v2.2.0
+	github.com/container-storage-interface/spec v1.4.0
+	github.com/golang/protobuf v1.4.3
+	github.com/stretchr/testify v1.6.1
+	golang.org/x/net v0.0.0-20210224082022-3d97a244fca7
+	google.golang.org/grpc v1.37.0
+	k8s.io/api v0.21.1
+	k8s.io/apimachinery v0.21.2-rc.0 // indirect
+	k8s.io/client-go v0.21.1
+	k8s.io/component-base v0.21.1
+	k8s.io/klog/v2 v2.8.0
 )
 
-replace k8s.io/api => k8s.io/api v0.19.0
+replace k8s.io/api => k8s.io/api v0.21.1
 
-replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.19.0
+replace k8s.io/apimachinery => k8s.io/apimachinery v0.21.2-rc.0
 
-replace k8s.io/apimachinery => k8s.io/apimachinery v0.19.0
+replace k8s.io/client-go => k8s.io/client-go v0.21.1
 
-replace k8s.io/apiserver => k8s.io/apiserver v0.19.0
-
-replace k8s.io/cli-runtime => k8s.io/cli-runtime v0.19.0
-
-replace k8s.io/client-go => k8s.io/client-go v0.19.0
-
-replace k8s.io/cloud-provider => k8s.io/cloud-provider v0.19.0
-
-replace k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.19.0
-
-replace k8s.io/code-generator => k8s.io/code-generator v0.19.0
-
-replace k8s.io/component-base => k8s.io/component-base v0.19.0
-
-replace k8s.io/cri-api => k8s.io/cri-api v0.19.0
-
-replace k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.19.0
-
-replace k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.19.0
-
-replace k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.19.0
-
-replace k8s.io/kube-proxy => k8s.io/kube-proxy v0.19.0
-
-replace k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.19.0
-
-replace k8s.io/kubectl => k8s.io/kubectl v0.19.0
-
-replace k8s.io/kubelet => k8s.io/kubelet v0.19.0
-
-replace k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.19.0
-
-replace k8s.io/metrics => k8s.io/metrics v0.19.0
+replace k8s.io/component-base => k8s.io/component-base v0.21.1
 
 replace k8s.io/node-api => k8s.io/node-api v0.19.0
```

Comment:
Why is node-api still v0.19.0?

Reply:
@pohly upgraded to 0.21.1 . Thanks! @pohly ptal.